src/log.h

```diff
@@ -25,32 +25,39 @@
 
 namespace rhost {
     namespace log {
+        enum class log_verbosity {
```

Comment:
Originally that's exactly what I did in managed, then discovered that LogLevel is already used for different things in host and also .NET Core uses the term for severity level. So I had to come up with something different. Oh well.

```
namespace Microsoft.Extensions.Logging {
    //
    // Summary:
    //     Defines logging severity levels.
    public enum LogLevel

```